source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,170 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah SLA AGM battery, capable of high momentary currents.
+
+SLA stands for Sealed Lead Acid, AGM stands for Absorbent Glass Mat.
+
+The Robot Battery assembly includes a COTS battery, with positive and negative lead cables to an Anderson SB connector. The Red/Pink Anderson SB50 is STRONGLY recommended.
+
+Always start your match with a fully charged battery - ask for help if it looks like you won't!
```

Comment:
I don't like the jumping around. Content needs to build off of itself. Either turn it into a bullet list, or reword the content to build off the previous statements. As it stands right now, each of these paragraphs is completely separate.

Reply:
I agree I would move "The Robot Batter.." sentance to the Battery Leads section and the "Always start..." to best practices.

Reply:
Brought this into a single paragraph